nntrainer/layers/layer_devel.h

```diff
@@ -50,6 +50,85 @@ enum class ExportMethods;
 class Layer {
 
 public:
+  /**
+   * @brief     Property Enumeration
+   *            0. input shape : string
+   *            1. bias zero : bool
```

Comment:
from 1 to 9 documentation doesn't match. (bias zero should be move to 9)

Reply:
updated.

Reply:
good catch.